controllers/scheduleController.js

```diff
@@ -3,8 +3,35 @@ const { getSchedule } = require('../utils')
 
 exports.getRoomSchedule = async (req, res) => {
   try {
-    return res.status(HttpStatus.OK).json(req.query.r)
-  } catch (error) {}
+    const orar = await getSchedule('./data/schedule.json')
+    if (req.query.r === undefined) {
+      return res.status(HttpStatus.OK).send(orar) // ya gettin all the rooms, man
+    }
+    const rooms = req.query.r.split(',')
+    console.log(rooms)
+    const result = {}
+    for (const an in orar) {
+      for (const zi in orar[an]) {
+        for (const entry in orar[an][zi]) {
+          for (const room in rooms) {
```

Comment:
this will return an error if r is just a string, and forof should be used for arrays.